selectel/provider.go

```diff
@@ -108,12 +136,24 @@ func Provider() *schema.Provider {
 }
 
 func configureProvider(_ context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
-	config := Config{
-		Token:    d.Get("token").(string),
-		Endpoint: d.Get("endpoint").(string),
+	var config Config
+	if v, ok := d.GetOk("token"); ok {
+		config.Token = v.(string)
+	}
+	if v, ok := d.GetOk("user"); ok {
+		config.User = v.(string)
+	}
+	if v, ok := d.GetOk("password"); ok {
+		config.Password = v.(string)
+	}
+	if v, ok := d.GetOk("endpoint"); ok {
+		config.Endpoint = v.(string)
+	}
+	if v, ok := d.GetOk("auth_url"); ok {
+		config.IdentityEndpoint = v.(string)
 	}
-	if v, ok := d.GetOk("project_id"); ok {
-		config.ProjectID = v.(string)
+	if v, ok := d.GetOk("domain_name"); ok {
+		config.DomainName = v.(string)
 	}
```

Comment:
Please give me a hint why we don't check `project_id` here anymore?

Reply:
mistake, fixed